src/pages/ProductList/components/Product.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { FaRegHeart } from 'react-icons/fa';
+import './Product.scss';
+
+export default function Product(props) {
+  const { url, name, price } = props;
+
+  return (
+    <div>
+      <div className="imageHolder">
+        <img alt={`${name}`} src={url} />
```

Comment:
특별히 string과 JS가 합쳐져 사용되지 않기 때문에 아래와 같이 작성해도 되겠네요!
```suggestion
        <img alt={name} src={url} />
```